### PR TITLE
Vis_Fixation plugin and added it to Player

### PR DESCRIPTION
**Pull request by snleu**

### CODE REVIEW

File: pupil_src/player/vis_fixation.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+from plugin import Plugin
+import numpy as np
+import cv2
+
+# TODO: Import pyglui
+from pyglui import ui
+
+from methods import denormalize
+
+from fixation_detector import Fixation_Detector
```
Comment (papr) [inline pupil_src/player/vis_fixation.py]: This import is not used in the code below.

File: pupil_src/player/vis_fixation.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ License details are in the file license.txt, distributed as part of this software.
+----------------------------------------------------------------------------------~(*)
+'''
+
+from player_methods import transparent_circle
+from plugin import Plugin
+import numpy as np
+import cv2
+
+# TODO: Import pyglui
```
Comment (papr) [inline pupil_src/player/vis_fixation.py]: TODO is resolved. See L17

Comment (snleu) [inline pupil_src/player/vis_fixation.py]: this was a copied from vis_circle, i'll delete it

File: pupil_src/player/vis_fixation.py (hunk 1 above)
Comment (snleu) [inline pupil_src/player/vis_fixation.py]: correct forgot I added this, i'll delete this as well

**Comment by mkassner** [comment]
Commit looks good! Please rebase onto current master so I can merge.

**Pull request closed without merging**